### PR TITLE
F/690-improve-equalizer

### LINKED ISSUE #690: Improve equalizer
**Is your feature request related to a problem? Please describe:**  
The tab for the equalizer is not self descriptive. The name and the numbers are not easy to understand.

**Please describe the solution you'd like:**  
Some changes improve the comprehensibility of the equalizer
- [ ] Change the name "Equalizer" (@Bensk1, @Bouncner: What is a good name for the "Equalizer"?)
- [ ] Add explanations for the meaning of the numbers (querie numbers and weights)

**Affected Component(s):**  
`Frontend`

**Additional context:**  
MultipleWorkloads3

### PR DESCRIPTION
**Pull request by kathwill**
# Resolves #690

**Does your pull request solve a problem? Please describe:**  
The tab for the equalizer is not self descriptive. The pull request changes the name of the tab and adds explanations for the shown numbers.

**Affected Component(s):**  
`Frontend`

**Additional context:**  
QueryWeights

QueryWeights3

### CODE REVIEW

**Comment by kathwill** [comment]
What do you think of this option? @Bensk1 @Bouncner

**Comment by Bensk1** [comment]
We are getting there!

- I am not a UI expert but what about putting the descriptions _query numbers_ and _query weights_ on the left side?
- Could we also rename _Query numbers_ to _Query ID_ and _Query Weights_ to _Query Weight_?
- Also, while we are on it. Do you think that a reset button (including functionality) is complicated?
- Does anyone have other UI/UX ideas? A different font/color to better differentiate between text labels/control elements?

**Comment by Bouncner** [comment]
Is it possible to change the input fields style to make it more obvious that this is an editable field and not just text?

**Comment by kathwill** [comment]
I renamed _Query numbers_ to _Query ID_ and _Query Weights_ to _Query Weight_ and added a reset button (as you can see in the pictures above).

> I am not a UI expert but what about putting the descriptions _query numbers_ and _query weights_ on the left side?

I decided to put it on the right side because I think if we put it on the left side it is too much information on that side because of the scale. What do you think of this @Bensk1?

**Comment by Bensk1** [comment]
> I decided to put it on the right side because I think if we put it on the left side it is too much information on that side because of the scale. What do you think of this @Bensk1?

Perfect, I like that better.

File: hyrisecockpit/frontend/src/components/workload/QueryWeights.vue [hunk 1]
```diff
@@ -52,8 +52,41 @@
...
             </div>
+            <div
+              v-if="
+                idx % numberOfQueriesPerRow === numberOfQueriesPerRow - 1 ||
+                idx ===
+                  numberOfQueriesPerWorkload[
+                    selectedWorkloads.indexOf(workload)
+                  ] -
+                    1
+              "
```
Comment (schTi) [inline hyrisecockpit/frontend/src/components/workload/QueryWeights.vue]: looks like this logic can be outsourced in an own function

File: hyrisecockpit/frontend/src/components/workload/QueryWeights.vue [hunk 2]
```diff
@@ -129,6 +167,15 @@ export default defineComponent({
       }
       context.emit("change", workload, name, weight.value);
     }
+    function resetWeights(workload: Workload): void {
+      Object.values(weights.value[workload]).forEach((weight) => {
+        if (weight.value !== 1) {
+          weight.value = 1;
+          weight.sliderValue = 50;
```
Comment (schTi) [inline hyrisecockpit/frontend/src/components/workload/QueryWeights.vue:168]: can you add a comment, why 50 is the base value?

**Review by schTi — CHANGES_REQUESTED** [review]

**Review by schTi — APPROVED** [review]

**Pull request merged**